### PR TITLE
 save operators iib data locally and in s3 bucket

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Comment by redhat-qe-bot1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest tox`: Retest tox
 * `/retest build-container`: Retest build-container

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: ci_jobs_trigger/libs/operators_iib_trigger/iib_trigger.py [hunk 1]
```diff
@@ -96,66 +114,87 @@ def get_new_iib(config_data, logger):
...
-                logger.info(f"{LOG_PREFIX} Committing changes for {OPERATORS_DATA_FILE_NAME}")
-                _git_repo.git.commit("-m", f"'Auto update: {OPERATORS_DATA_FILE_NAME}'")
-                logger.info(f"{LOG_PREFIX} Push new changes for {OPERATORS_DATA_FILE}")
-                _git_repo.git.push(repo_url)
-                logger.info(f"{LOG_PREFIX} New changes for {OPERATORS_DATA_FILE_NAME} pushed")
-        except Exception as ex:
-            err_msg = f"{LOG_PREFIX} Failed to update {OPERATORS_DATA_FILE_NAME}. {ex}"
-            logger.error(err_msg)
-            send_slack_message(message=err_msg, webhook_url=slack_webhook_url, logger=logger)
+    except Exception as ex:
```
Comment (myakove) [inline ci_jobs_trigger/libs/operators_iib_trigger/iib_trigger.py:150]: Missing `LOG_PREFIX` in the log error message.

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Comment by redhat-qe-bot2** [comment]
New container for quay.io/redhat_msi/ci-jobs-trigger:latest published

**Pull request merged**